src/libraries/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -94,7 +94,7 @@ public void PathAlreadyExistsAsDirectory(FileAttributes attributes)
             }
             finally
             {
-                testDir.Attributes = original;
+                testDir.Attributes = FileAttributes.Normal;
```

Comment:
Can you please elaborate on why this is needed?

Reply:
the new directory has FileAttributes.Normal, there's no need to read them.

if you mean - why bother setting them back - it's because a r/o directory can't be deleted. this was before I went in an added code in the cleanup to handle this, but it seems reasonable cleanup here.

Reply:
I mean, before we were restoring `Attributes` to its `original` value, I don't see how `original` could contain the `ReadOnly` flag if the directory was just created one line above.

Reply:
It couldn't, but it was an extra line of code.

Reply:
Oh, I see your point. In this case I guess I did it only for cleanup - remove the extra line.